parastell/parastell.py

```diff
@@ -622,10 +660,16 @@ def parastell():
     
     if args.ivb or args.magnets:
         dagmc_export = all_data['dagmc_export']
+        stellarator.build_cubit_model(
+            **(filter_kwargs(dagmc_export, build_cubit_model_allowed_kwargs)))
```

Comment:
Use proper formatting.
```suggestion
        stellarator.build_cubit_model(
            **(filter_kwargs(dagmc_export, build_cubit_model_allowed_kwargs))
        )
```